packages/shared/src/server/clickhouse/schema.ts

```diff
@@ -2,6 +2,11 @@ export const ClickhouseTableNames = {
   traces: "traces",
   observations: "observations",
   scores: "scores",
+
+  // Virtual tables for dashboards
+  // TODO: Check if we can do this more elegantly
+  scores_numeric: "scores_numeric",
+  scores_categorical: "scores_categorical",
```

Comment:
@maxdeichmann What do you think about this approach to support the new "virtual" views? Do you see any risks if we allow table names here that don't exist physically?

Those get used within the FilterState logic that I reuse for the dashboards.